go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/trufflesecurity/trufflehog/v3
 
-go 1.21
+go 1.22
```

Comment:
uhh what? lol

Reply:
I'll upgrade CI and upstream to 1.22 before bumping this. It's using the new rand libraries so requires 1.22